test/workers_test.rb

```diff
@@ -194,87 +155,4 @@ def test_empty_services
     sleep(2 * FLUSH_INTERVAL)
     assert_equal(0, @writer.stats[:services_flushed], 'wrong number of services flushed')
   end
-
-  # test that services are correctly flushed, with two of them
-  def test_two_services
```

Comment:
This one should be deleted? if we remove it, we don't have any test to check if services are properly flushed by the worker. Can we include it or a variant that isn't impacted by the old retry policy?